index.js

```diff
@@ -5,21 +5,21 @@
 'use strict';
 
 function warn() {
-    if ('production' !== process.env.NODE_ENV) {
-        console.warn('Warning: the function is client-side only, does not work on server side');
-    }
+  if ('production' !== process.env.NODE_ENV) {
+    console.warn('Warning: the function is client-side only, does not work on server side');
+  }
 }
 
 if (typeof window !== 'undefined') {
-    module.exports = {
-        listen: require('./dist/lib/listen'),
-        subscribe: require('./dist/subscribe'),
-        unsubscribe: require('./dist/unsubscribe')
-    };
+  export default {
```

Comment:
Ideally these would be separate exports so that webpack could tree shake the libraries that are not used. I believe we typically `require` the individual files anyways.

Addtionally, you could create a new `module.js` file that exports each library separately and then gets referenced in the `package.json` -> `module` property. But I believe that would change the API for consumers that consume the `index.js` file.

Reply:
sure, at first I was thinking about not to introduce breaking changes, so I keep the object export there, but as we are not touching this library for a while, probably just change it all once for tree sharking

Reply:
I looked internally and we only have a handful of packages using this, so its easy to update the major version. So I'm all for doing the refactor now.